examples/vertx-opentracing/pom.xml

```diff
@@ -22,7 +22,7 @@
 
   <groupId>org.hawkular.apm</groupId>
   <artifactId>hawkular-apm-examples-vertx-opentracing</artifactId>
-  <version>0.1-SNAPSHOT</version>
+  <version>0.11.1.Final-SNAPSHOT</version>
```

Comment:
It can't reference the parent, as when the docker image is being built it complains. I tried specifying parent relative path before and it didn't work, so had to resort to the parent pom including it as a module (so gets included in the builds and should be released correctly), but still enable the docker build to work.